### PR TITLE
Upgrade node

### PR DESCRIPTION
**Pull request by Sreejith-Websphere**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: scripts/build/docs_part_1.sh [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+
 set -e
 export BUILD_SCRIPTS_DIR=$(dirname $0)
 echo "BUILD_SCRIPTS_DIR: $BUILD_SCRIPTS_DIR"
 
 #Antora Portion of Docs
 echo "Begin building of Antora portion of docs"
 
-$BUILD_SCRIPTS_DIR/node_install.sh
+#$BUILD_SCRIPTS_DIR/node_install.sh
```
Comment (kinueng) [inline scripts/build/docs_part_1.sh]: Are we effectively saying with this PR that we no longer need the `node_install.sh` file in our automated builds?

I do see that we call `node_install.sh` in other bash scripts, but I think those scripts are for local development only.

Comment (Sreejith-Websphere) [reply scripts/build/docs_part_1.sh]: yes you are right for our automated build we not need to call node install file. The file it calls node install file its for local dev

Comment (kinueng) [reply scripts/build/docs_part_1.sh]:
```suggestion
```

I would suggest deleting the line over commenting it out

Comment (Sreejith-Websphere) [reply scripts/build/docs_part_1.sh]: ok

**Review by aknguyen7 — APPROVED** [review]

**Pull request merged**